test/string-replace-all/index.js

```diff
@@ -0,0 +1,24 @@
+var test = require('../util/test')(__filename);
+var replaceAll = require('../../packages/string-replace-all');
+
+test('string with replacements', function(t) {
+  t.plan(3);
+  t.equal(replaceAll('hello, world', 'l', 'qq'), 'heqqqqo, worqqd');
+  t.equal(replaceAll('hello, world', 'll', 'q'), 'heqo, world');
+  t.equal(replaceAll('hello, world', 'l', ''), 'heo, word');
+  t.end();
+});
+
+test('string with no replacements', function(t) {
+  t.plan(3);
+  t.equal(replaceAll('hello, world', '', 'q'), 'hello, world');
+  t.equal(replaceAll('hello, world', null, 'q'), 'hello, world');
+  t.equal(replaceAll('hello, world', 'l', null), 'hello, world');
+  t.end();
+});
+
+test('string with null,', function(t) {
```

Comment:
Test that a non-string on each argument position throws. Also that < 3 arguments throws

e.g.
```
t.throws(function() {
    replaceAll(null, 'hi', 'bye');
  });
```